test_cases/deploy_time_variables/out/DeployTimeVariablesContract.approval.teal

```diff
@@ -0,0 +1,87 @@
+#pragma version 10
+
+test_cases.deploy_time_variables.contract.DeployTimeVariablesContract.approval_program:
+    // deploy_time_variables/contract.py:4
+    // class DeployTimeVariablesContract(arc4.ARC4Contract):
+    txn NumAppArgs
+    bz main_bare_routing@6
+    method "get_bytes()byte[]"
+    method "get_uint()uint64"
+    txna ApplicationArgs 0
+    match main_get_bytes_route@2 main_get_uint_route@3
+    err // reject transaction
+
+main_get_bytes_route@2:
+    // deploy_time_variables/contract.py:5
+    // @arc4.abimethod()
+    txn OnCompletion
+    !
+    assert // OnCompletion is NoOp
+    txn ApplicationID
+    assert // is not creating
+    callsub get_bytes
+    dup
+    len
+    itob
+    substring 6 8
+    swap
+    concat
+    byte 0x151f7c75
+    swap
+    concat
+    log
+    int 1
+    return
+
+main_get_uint_route@3:
+    // deploy_time_variables/contract.py:9
+    // @arc4.abimethod()
+    txn OnCompletion
+    !
+    assert // OnCompletion is NoOp
+    txn ApplicationID
+    assert // is not creating
+    callsub get_uint
+    itob
+    byte 0x151f7c75
+    swap
+    concat
+    log
+    int 1
+    return
+
+main_bare_routing@6:
+    // deploy_time_variables/contract.py:4
+    // class DeployTimeVariablesContract(arc4.ARC4Contract):
+    txn OnCompletion
+    !
+    assert // reject transaction
+    txn ApplicationID
+    !
+    assert // is creating
+    int 1
+    return
+
+
+// test_cases.deploy_time_variables.contract.DeployTimeVariablesContract.get_bytes() -> bytes:
+get_bytes:
+    // deploy_time_variables/contract.py:5-6
+    // @arc4.abimethod()
+    // def get_bytes(self) -> Bytes:
+    proto 0 1
+    // deploy_time_variables/contract.py:7
+    // return BytesDeployVar("SOME_BYTES")
+    pushbyte TMPL_SOME_BYTES
+    retsub
+
+
+// test_cases.deploy_time_variables.contract.DeployTimeVariablesContract.get_uint() -> uint64:
+get_uint:
+    // deploy_time_variables/contract.py:9-10
+    // @arc4.abimethod()
+    // def get_uint(self) -> UInt64:
+    proto 0 1
+    // deploy_time_variables/contract.py:11
+    // return UInt64DeployVar("SOME_UINT")
+    pushint TMPL_SOME_UINT
```

Comment:
Beaker includes a comment with the variable name after it. We probably don't need that if we're including source comments?

```
pushint TMPL_DELETABLE // TMPL_DELETABLE
```